packages/react/src/components/IonPopover.tsx

```diff
@@ -1,9 +1,8 @@
-import { JSX } from '@ionic/core/components';
-import { IonPopover as IonPopoverCmp } from '@ionic/core/components/ion-popover.js';
+import { JSX, popoverController } from '@ionic/core/components';
 
 import { createInlineOverlayComponent } from './createInlineOverlayComponent'
 
 export const IonPopover = /*@__PURE__*/ createInlineOverlayComponent<
   JSX.IonPopover,
   HTMLIonPopoverElement
->('ion-popover', IonPopoverCmp);
```

Comment:
If you import `defineCustomElements` instead and call that function, it should define both the overlay and the backdrop. See https://github.com/ionic-team/ionic-framework/blob/main/packages/vue/src/components/IonApp.ts#L3-L7 as an example